packages/babel-preset-mapbox/README.md

```diff
@@ -4,82 +4,124 @@ A configurable Babel preset which abstracts away the most common Babel plugins a
+- [@babel/plugin-transform-runtime](https://babeljs.io/docs/en/babel-plugin-transform-runtime)
+- [@babel/plugin-syntax-dynamic-import](https://babeljs.io/docs/en/babel-plugin-syntax-dynamic-import)
+- [babel-plugin-transform-dynamic-import](https://www.npmjs.com/package/babel-plugin-transform-dynamic-import)
 - [babel-plugin-transform-react-remove-prop-types](https://www.npmjs.com/package/babel-plugin-transform-react-remove-prop-types)
-- [babel-plugin-transform-runtime](https://www.npmjs.com/package/babel-plugin-transform-runtime)
-- [babel-preset-env](https://www.npmjs.com/package/babel-preset-env)
-- [babel-preset-react](https://www.npmjs.com/package/babel-preset-react)
 
 ### Modifying browser support with the help of `browserslist`
 
 You can customize the amount of compilation done by giving a list of browsers you want to support. We use [browserslist](https://github.com/ai/browserslist) to parse this information, so you can use [any valid query format supported by browserslist](https://github.com/ai/browserslist#queries).
 
-`@mapbox/babel-preset-mapbox` allows you to customize [browserslist](https://github.com/ai/browserslist) by setting the environment variable `BROWSERSLIST`. You can set it directly in your shell before running Babel:
+`@mapbox/babel-preset-mapbox` allows you to customize [browserslist](https://github.com/ai/browserslist) by using any of the methods mentioned [here](https://github.com/browserslist/browserslist#packagejson).
+
+An example of setting setting the environment variable `BROWSERSLIST`:
 
 ```bash
 export BROWSERSLIST=">0.25%, not ie 11"
 npx babel script.js
 ```
 
-Make sure your `.babelrc` files includes the `@mapbox/babel-preset-mapbox` preset:
-
-```
-// .babelrc
-{
-  "presets": [
-    "@mapbox/babel-preset-mapbox"
-  ]
-}
-```
-
 You can also set the `BROWSERSLIST` environment variable in your node application before parsing code with Babel and `"@mapbox/babel-preset-mapbox"`:
 
 ```js
 const babel = require('babel-core');
+const fs = require('fs');
+
+process.env.BROWSERSLIST = '>0.25%, not ie 11';
 
-process.env.BROWSERSLIST = ">0.25%, not ie 11";
+const srcCode = fs.readFileSync('./src/index.js', 'utf8');
 
-babel.transform(code, {
-    presets: [require.resolve('@mapbox/babel-preset-mapbox')],
-    filename: 'source.js'
+const output = babel.transform(srcCode, {
+  presets: [require.resolve('@mapbox/babel-preset-mapbox')],
+  filename: 'source.js'
 }).code;
+
+console.log(output);
 ```
 
-**Note:** Please avoid using [babel-preset-env](https://www.npmjs.com/package/babel-preset-env) to define your browser support. Using it would throw an **error** to avoid conflicting with the `BROWSERSLIST` env var. You should instead just set the value of environment variable `BROWSERSLIST`. If your use case doesn't allow for this, please feel free to open a ticket about it.
+### Advanced configuration
+
+You can pass a callback `override` option to fully customize this preset:
```

Comment:
Are there any Babel docs about this that we can link to?

Reply:
Not sure if there is anything particular for passing a callback to a preset.